journal.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import os
+from pyramid.config import Configurator
+from pyramid.view import view_config
+from waitress import serve
+import sqlalchemy as sa
+from sqlalchemy.ext.declarative import declarative_base
+import datetime
+
+
+Base = declarative_base()
+
+
+class Entry(Base):
+
+    __tablename__ = "entries"
+
+    id = sa.Column(sa.Integer, primary_key=True, autoincrement=True)
+    title = sa.Column(sa.Unicode(255), nullable=False)
+    creation_date = sa.Column(sa.DateTime, nullable=False, default=datetime.datetime.utcnow)
+    entry_text = sa.Column(sa.UnicodeText, nullable=False)
+
+    def __repr__(self):
+        return "<Entry(title='%s', creation_date='%s')>" % (self.title, self.creation_date)
```

Comment:
Using string.format() is more forward compatible than the `%s` string templating approach you use here.